docs/source/ecf.rst

```diff
@@ -240,30 +271,79 @@ The names of the the files needed are given in the header as ``hdulist[0].header
 Stage 4
 --------
 
-
-.. include:: S4_template.ecf
+.. include:: ../media/S4_template.ecf
    :literal:
 
 nspecchan
-'''''''''''
-
+''''''''''
 Number of spectroscopic channels spread evenly over given wavelength range
 
 
 wave_min & wave_max
-'''''''''''''''''''''''
-
+''''''''''''''''''''
 Start and End of the wavelength range being considered
 
 
+allapers
+'''''''''
+If True, run S4 on all of the apertures considered in S3. Otherwise will use the only or newest S3 outputs found in the inputdir
```

Comment:
How about the following:

If True, run S4 on all of the apertures considered in S3. Otherwise the code will use the only or newest S3 outputs found in the inputdir. To specify a particular S3 save file, ensure that "outputdir" points to the procedurally generated folder containing that save file (e.g. set outputdir to /Data/JWST-Sim/NIRCam/Stage3/S3_2021-11-08_nircam_wfss_ap10_bg10_run1/).

Reply:
In thinking more about this, the results from running S3 with each aperture/bg size are saved into individual folders, so there will never be an instance where multiple save files are located in the same directory.  If `allapers == True`, how do we efficiently track down all of the S3 save files in different directories?

Reply:
There's two things I want to clarify here. The first is that I typically don't specify the inputdir to that level of precision (aka I set inputdir to just /Data/JWST-Sim/NIRCam/Stage3/, not /Data/JWST-Sim/NIRCam/Stage3/S3_2021-11-08_nircam_wfss_ap10_bg10_run1/). This means I don't need to update my ecf every single time I run the code since it will just use the newest one, but if I want to consider a specific run then you can give that level of precision in your inputdir variable. Second, the code uses the metadata file to figure out what folders the other aperture pairs would be located in by reusing the same code that created those different folders in the first place.

Reply:
The code that does this job of locating the other aperture folders is on this line: https://github.com/kevin218/Eureka/blob/a9f683bd7fc79d6fe3f2ba8debc334985ab18d1a/eureka/S4_generate_lightcurves/s4_genLC.py#L141

Reply:
Ok, this makes a lot more sense.  I'm good with the proposed changes.